### DISCUSSION TITLE
Utilities permissions missing since 3.4.5

### DISCUSSION THREAD

**Discussion by tidygraphic** [statamic/cms#7795] [Troubleshooting] [answered]
Since updating to 3.4.5 we seem to have lost the ability to allow users Utilities permissions (specifically access to reset the cache). It appears this block of permissions is missing: 

<img width="983" alt="Screenshot 2023-03-28 at 15 46 47" src="https://user-images.githubusercontent.com/5737710/228276681-d015c402-acef-4af1-8f59-82664b16f871.png">

Is this a bug or expected? Can I allow users access to this now?

Thanks, Lea.

**Comment by joshuablum** [comment, marked as answer]
Hey @tidygraphic,

I just tried it with a fresh Statamic install using v3.4.5 and wasn't able to replicate the issue. Works perfectly fine for me.

Could you provide some more details?
- Like the output of `php please support:details`
- Are you logged in as a super user?
- From which version did you upgrade?
![CleanShot 2023-03-29 at 2 13 06](https://user-images.githubusercontent.com/3824203/228531941-c2c7c31f-c87a-422b-bdfb-f1f0eea4581a.png)

**Reply by tidygraphic** [reply]
That's strange! Is there anything we can do to 'flush' the permission then or maybe upgrading Statamic again will fix this. 

## Output of support:details

```
Environment
Application Name: ASI
Laravel Version: 9.52.4
PHP Version: 8.1.10
Composer Version: 2.0.10
Environment: production
Debug Mode: OFF
Maintenance Mode: OFF

Cache
Config: CACHED
Events: NOT CACHED
Routes: CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 1
Antlers: regex
Stache Watcher: Enabled
Static Caching: half
Version: 3.4.5 PRO

Statamic Addons
rootstudio/monitor: 0.1
```

I'm logged in as a super user and we last moved from `v3.4.3` but I can't be sure when the problem actually appear (we've only just been asked to add the option from a customer).

**Reply by joshuablum** [reply]
Hm, try upgrading to the latest version (`v3.4.7`) and run the following commands:
- `php artisan optimize:clear`
- `php please stache:clear`
- `php artisan vendor:publish --tag=statamic-cp`

Also try a hard-refresh to clear the cache on that site.
Loggin in as a different (super) user results in the same issue?

**Comment by tidygraphic** [comment]
Thanks @joshuablum one of those steps has fixed the issue for one affected site and so we'll roll this out to all sites affected.

Thanks for your help!
